spec/rubocop/cop/root_cops/avoid_ruby_prof_spec.rb

```diff
@@ -9,7 +9,7 @@
 
   context "when ruby_prof is the only metadata used in a `describe` block's definition" do
     it "reports an offense" do
-      expect_offense(<<~RUBY.strip_indent)
```

Comment:
These are new cops enabled by default?

What was `strip_indent` doing that the `<<~` wasn't already?

Reply:
IIRC `strip_indent` was primarily useful for `<<-`. `<<~` does this for you 🤷‍♂